src/checker.py

```diff
@@ -18,7 +20,7 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 from collections import OrderedDict
-from lib.externaldata import CheckerRegistry, ExternalData
+from lib import CheckerRegistry, CVEData, ExternalData
```

Comment:
I think this change is out of scope. I don't know how this project will evolve so I prefer to keep things separated like I did.